templates/security/esm.html

```diff
@@ -134,19 +162,73 @@ <h4 class="u-no-max-width">Do all levels of Ubuntu Advantage for Infrastructure
       <p>Yes. Ubuntu ESM is available for UA-I Essential, Standard and Advanced customers. For more information on levels please visit our <a href="/pricing">pricing page</a>. Existing UA customers can retrieve their credentials through the <a href="/advantage">Ubuntu Advantage portal</a>.</p>
 
       <h4 class="u-no-max-width">How can we ensure the security of our Ubuntu systems after the end of Standard Security Maintenance?</h4>
-      <p>Sign up to Ubuntu Advantage now, and you will benefit from UA services immediately without having a gap in service when a particular release reaches the end of its Standard Security Maintenance window. For more details of initial release dates and maintenance cadence, see <a href="https://wiki.ubuntu.com/Releases">wiki.ubuntu.com/Releases</a>. Ubuntu Advantage is available through the <a href="/advantage">Ubuntu Advantage portal</a> and through the <a class="p-link--external" href="https://aws.amazon.com/marketplace/pp/B01MZ1LSBQ?qid=1486382973569&sr=0-1&ref_=srh_res_product_title">AWS Marketplace</a>.</p>
+      <p>
+        To ensure security continuity when a particular release reaches the end of its Standard Security Maintenance window, sign up for <a href="/advantage">Ubuntu Advantage</a> for free on up to 3 machines or through a paid subscription for enterprise use. For more details of initial release dates and maintenance cadence, see <a href="https://ubuntu.com/about/release-cycle">https://ubuntu.com/about/release-cycle</a>.
```

Comment:
Also left another comment about this as we don't normally link to pages like this, I think it should be something like "see Ubuntu's release cycle" or  something similar but we can wait for a response from Lech.